.travis.yml

```diff
@@ -30,7 +30,7 @@ notifications:
 
 env:
   global:
-   - MAVEN_OVERRIDE="--settings=.travis/settings.xml -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true"
+   - MAVEN_OVERRIDE="--settings=.travis/settings.xml"
```

Comment:
That's optimizing for the wrong use case. We need `-Pprecommit` or something like that for "slow mode".

(Just like internally, the default behavior is to run tests, and we don't block your running tests on lint passing.)